### PR TITLE
Jira 123 update readme

### PR DESCRIPTION
**Pull request by kunalmenaria**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -76,3 +76,4 @@ Inspired by
 * [Mojang Authlib](https://github.com/Techcable/Authlib)
 * [Mojang AccountsClient](https://github.com/JonMcPherson/AccountsClient/)
 * [API proxy Interceptor](https://github.com/Shevchik/MojangAPIProxy)
+hi, kunal here syystummm
```
Comment (jaydeepdangi) [inline README.md]: dont be a fanboy

**Comment by Anuragmenaria** [comment]
hii darling

**Comment by kunalmenaria** [comment]
kya jii

**Comment by kunalmenaria** [comment]
mind your bussiness

**Comment by kunalmenaria** [comment]
i know spelling is wrong , but do your work

**Comment by Anuragmenaria** [comment]
OK Buddy

**Pull request merged**